test/testcontracts.js

```diff
@@ -8,16 +11,34 @@ contract("CaptureTheFlag", async accounts => {
     let captureFlagContract
 
     before(async () => {
-        captureFlagContract = await CaptureTheFlag.new();
+        const {forwarderAddress, paymasterAddress} = GsnTestEnvironment.loadDeployment()
+
+        captureFlagContract = await CaptureTheFlag.new(forwarderAddress);
+
+        const gsnProvider = await RelayProvider.newProvider({
+            provider: web3.currentProvider,
```

Comment:
where is the `web3` object coming from?

Reply:
@drortirosh

Reply:
In this project the tests are executed by a Truffle (https://trufflesuite.com/truffle/) 
It injects some global variables to your test environment, including the `web3` object.